packages/gatsby/package.json

```diff
@@ -79,12 +80,14 @@
     "gatsby-plugin-typescript": "^2.4.2",
     "gatsby-react-router-scroll": "^3.0.0",
     "gatsby-telemetry": "^1.3.5",
+    "get-port": "^5.1.1",
```

Comment:
for other things, we've used [`detect-port`](https://github.com/node-modules/detect-port). Maybe use the same?